tests/suites/smoke/deploy.sh

```diff
@@ -16,8 +16,9 @@ run_local_deploy() {
 	juju refresh refresher
 
 	# Wait for the refresh to happen and then wait again.
-	wait_for "upgrade hook ran v2" "$(workloadstatus "refresher" 0)"
+	wait_for "upgrade hook ran v2" "$(workload_status "refresher" 0)"
 
+	juju remove-application refresher
```

Comment:
Probably leave a TODO for removing in the future.

Reply:
I don't see the issue. It's probably good practice to remove the application before tearing down the model anyway

Reply:
This is definitely a bug, so removing app before teardown is a temp fix.
We should remove the temp fix once we fix the teardown issue.